src/main/proto/th2_grpc_check1/check1.proto

```diff
@@ -132,4 +134,27 @@ message WaitForResultRequest {
 message WaitForResultResponse {
     RequestStatus status = 1;
     EventStatus rule_result = 2;
+}
+
+message MultiRulesRequest {
+    ChainID default_chain = 1;
```

Comment:
Could we declare the common parent_event_id with the same logic:
each rule without parent event id takes the parent_event_id from rule request.